### PR TITLE
feat(docs): update check results page with retries [sc-14874]

### PR DESCRIPTION
**Pull request by miliberlin**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

I'll add the screenshots once we have meaningful data.

### CODE REVIEW

File: site/content/docs/monitoring/check-results.md [hunk 1]
```diff
@@ -112,4 +119,15 @@ These include:
...
+    e. Download
+
+
+## Check results with retries
+
+When retries are enabled through the "Double-check on failure" setting, tabs will indicate that the check result contains multiple check runs:
+1. The initial failed attempt
+2. The final result (which may have failed or succeeded)
+
+By selecting a check run all data and assets are available for inspection for each attempt.
```
Comment (adamwardecki) [inline site/content/docs/monitoring/check-results.md:131]:
```suggestion
When selecting a check run, all data and assets are available for inspection for each attempt.
```

**Review by adamwardecki — CHANGES_REQUESTED** [review]
One nit

File: site/content/docs/monitoring/check-results.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Select a check and you will see a breakdown of its recent runs, together with ke
 
 ### Summary section
 
-The summary at the top of the page allows for filtering based all of the page's data points based on the selected timeframe and locations. 
+The summary at the top of the page allows for filtering based on all of the page's data points and the selected timeframe and locations. Retried check runs do not influence this section and only final results are taken into account.
```
Comment (drakirnosslin) [inline site/content/docs/monitoring/check-results.md:19]: Suggestion for conciseness. 
```suggestion
The summary at the top of the page allows for filtering based on the page's data points and the selected timeframe and locations. Retried check runs do not influence this section; only the final results are considered.
```

File: site/content/docs/monitoring/check-results.md [hunk 3]
```diff
@@ -30,6 +30,13 @@ Single check runs can be accessed by selecting them on the time ribbon or by hov
 
 Selecting a check execution will take you to the dedicated check result page for it, which will look different based on whether you had been running a [Browser](#browser-check-results) or an [API check](#api-check-results).
 
+When retries are enabled, an additional icon highlights that the check result contains multiple check runs. All attempts as well as the final result are available to view.
+
+The check retries ratio indicates how many of the total checks run were executed twice due to an initial failed attempt.
```
Comment (drakirnosslin) [inline site/content/docs/monitoring/check-results.md:35]: I'm not sure I like this suggestion more TBH - it's just a way to get around having to update this again if we add the capability of having more than 1 retry per check run.
```suggestion
The check retries ratio indicates how many of the total checks run were retried due to an initial failed attempt.
```

File: site/content/docs/monitoring/check-results.md [hunk 4]
```diff
@@ -112,4 +119,15 @@ These include:
...
     d. First Byte
 
-    e. Download
\ No newline at end of file
+    e. Download
+
+
+## Check results with retries
+
+When retries are enabled through the "Double-check on failure" setting, tabs will indicate that the check result contains multiple check runs:
```
Comment (drakirnosslin) [inline site/content/docs/monitoring/check-results.md:127]:
```suggestion
When retries are enabled through the "[Double-check on failure](/docs/alerting/#double-checking)" setting, tabs will indicate that the check result contains multiple check runs:
```

**Review by drakirnosslin — APPROVED** [review]
Looks good 👍 
Added a few nitpicks, but they are minor.

**Review by drakirnosslin — DISMISSED** [review]

**Review by adamwardecki — COMMENTED** [review]
Looks like a good opportunity to update two other screenshots on this page.

**Review by drakirnosslin — DISMISSED** [review]

**Review by adamwardecki — APPROVED** [review]

**Pull request merged**